src/sas/qtgui/Plotting/PlotterBase.py

```diff
@@ -265,17 +267,25 @@ def onResize(self, event):
         """
         pass
 
-    def contextMenuEvent(self, event):
-        """
-        Display the context menu
-        """
+    def showContextMenu(self, point):
         if not self.quickplot:
             self.createContextMenu()
         else:
             self.createContextMenuQuick()
-
-        event_pos = event.pos()
-        self.contextMenu.exec_(self.canvas.mapToGlobal(event_pos))
+        # show the context menu at the specified point
+        self.contextMenu.exec_(self.mapToGlobal(point))
+
+    # def contextMenuEvent(self, event):
```

Comment:
No longer needed? Maybe get rid of it.

Reply:
This was left for introspection but I should at least add a comment.